src/pybind/mgr/dashboard/frontend/src/app/ceph/cephfs/cephfs-subvolume-form/cephfs-subvolume-form.component.html

```diff
@@ -43,6 +43,26 @@
           </div>
         </div>
 
+          <!--Subvolume Group name -->
+          <div *ngIf="!this.isEdit"
+               class="form-group row">
+            <label class="cd-col-form-label"
+                   for="subvolumeGroupName"
+                   i18n>Subvolume group name
+            </label>
+            <div class="cd-col-form-input">
+              <select class="form-select"
+                      id="subvolumeGroupName"
+                      name="subvolumeGroupName"
+                      formControlName="subvolumeGroupName">
+                <ng-container *ngIf="subVolumeGroups$ | async as subvolumeGroups">
+                  <option *ngFor="let subvolumegroup of subvolumeGroups"
+                          [value]="subvolumegroup.name">{{ subvolumegroup.name }}</option>
```

Comment:
If there are more than 1 subvolume groups, we should show something like `--- Select Subvolume group---` or something which is similar to what we have everywhere else.